docs/meetings/team/2022_09_27.md

```diff
@@ -0,0 +1,134 @@
+# 🚀 Weekly Team Meeting Minutes
+
+### **Start Time** 9:34pm
+
+### **End Time** 10:30
+
+### **Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Edouard
+3. Matt
+4. Logan
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad joke of the day
```

Comment:
Why do Vampires make poor project managers?
Because the refuse you to meet with stake holders.

Reply:
I like this one a lot, good dad joke 👍